temporalnexus/operation.go

```diff
@@ -324,12 +325,21 @@ func ExecuteUntypedWorkflow[R any](
 	if startWorkflowOptions.TaskQueue == "" {
 		startWorkflowOptions.TaskQueue = nctx.TaskQueue
 	}
+	if startWorkflowOptions.ID == "" {
```

Comment:
I'm wondering if we want to require setting workflow ID for Nexus operations. We require it in all Core based SDKs, maybe we can at least protect Nexus users. Not blocking this PR though. @cretz, @Quinn-With-Two-Ns to weigh in on this.

Reply:
I would change to ensure that the client doesn't auto generate an ID if one isn't set.

We should note this change in the release notes. I would consider it a backwards incompatible behavior change.

```go
	if startWorkflowOptions.ID == "" {
		return nil, ErrMissingWorkflowID
	}

```

Reply:
@pdoerner this already changed for Java, please make sure to disallow empty workflow ID for nexus ops before merging this.